java/buildconf/apidoc/asciidoc/scripts.txt

```diff
@@ -262,6 +262,28 @@ print(client.user.list_users(key))
 client.auth.logout(key)
 ----
 
+The following code shows how to use date-time parameters. This code will schedule immediate installation of package rhnlib-2.5.22.9.el6.noarch to system with id 1000000001.
+
+[source,python]
+----
+#!/usr/bin/env python3
+from datetime import datetime
+from xmlrpc.client import ServerProxy
+
+MANAGER_URL = "http://manager.example.com/rpc/api"
+MANAGER_LOGIN = "username"
+MANAGER_PASSWORD = "password"
+
+client = ServerProxy(MANAGER_URL)
+
+key = client.auth.login(MANAGER_LOGIN, MANAGER_PASSWORD)
+package_list = client.packages.findByNvrea(key, 'rhnlib', '2.5.22', '9.el6', '', 'noarch')
+earliest_occurrence = datetime.today()
+client.system.schedulePackageInstall(key, 1000000001, [package_list[0]['id']], earliest_occurrence)
```

Comment:
Have you tested this? There is an internal DateTime in xmlrpc which is used. Not sure if datetime.datetime is working out of the box.

Reply:
Yes, I have tested it and it is working fine.